### PR TITLE
updated iOS files for RN 0.61 compatibility & Tested app functionality for iOS.

### PR DESCRIPTION
**Pull request by josectobar**
**What does this implement/fix? Explain your changes.**
- Changed files in the Xcode project for React-native 0.61 compatibility. 

- Updated iOS podfile and podfile.lock related dependencies.
- Tested app in iOS.

**Screenshots**

IMG_8974
IMG_8975
IMG_8976
IMG_8977
IMG_8978
IMG_8979
IMG_8980
IMG_8981
IMG_8982
IMG_8983
IMG_8984
IMG_8985
IMG_8986

**Any other comments?**

- There seem to be some issues related to the react native vector icon library being used.

- The centered modal menu is shown blank. 

- In the detail screen from a product, the sizes and color selector are overflowing and overlapping in the screen.

**Where has this been tested?**
---------------------------
 - Magento Version: [e.g. 2.1.0]

 - Device: iPhone 7 with iOS 13 and iPhone6 with iOS 12

 - OS: iOS 13, iOS 12

### CODE REVIEW

**Comment by josectobar** [comment]
#63 , #40

**Comment by alexakasanjeev** [comment]
Thank you so much @josectobar for your help.

one thing, even after fixing material design icon, I can see 'react-native-vector-icon' is not loading, correctly, in place of icon, question mark is coming, there should be a humburger icon for drawer, and a cart icon in Home screen, could you please look into it.

react-native-vector-icon guide: https://github.com/oblador/react-native-vector-icons

Reference photo on Android, how icon looks

**Comment by josectobar** [comment]
@alexakasanjeev, I fixed the icons. 

IMG_8987
IMG_8988
IMG_8989
IMG_8990
IMG_8991
IMG_8992
IMG_8993

**Comment by alexakasanjeev** [comment]
Let me test the branch in Android Device at my end, and let me make sure everything is working, because there are some Android related file changed, will review your Pull request.

**Comment by josectobar** [comment]
Yeah, those got updated automatically.

File: android/.project [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+		<buildCommand>
+			<name>org.eclipse.buildship.core.gradleprojectbuilder</name>
+			<arguments>
+			</arguments>
+		</buildCommand>
+	</buildSpec>
+	<natures>
+		<nature>org.eclipse.buildship.core.gradleprojectnature</nature>
+	</natures>
+</projectDescription>
```
Comment (alexakasanjeev) [inline android/.project]: Delete this file

File: android/.settings/org.eclipse.buildship.core.prefs [hunk 2]
```diff
@@ -0,0 +1,2 @@
+connection.project.dir=
+eclipse.preferences.version=1
```
Comment (alexakasanjeev) [inline android/.settings/org.eclipse.buildship.core.prefs]: Delete this file

File: android/app/.classpath [hunk 3]
```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<classpath>
+	<classpathentry kind="con" path="org.eclipse.jdt.launching.JRE_CONTAINER/org.eclipse.jdt.internal.debug.ui.launcher.StandardVMType/JavaSE-1.8/"/>
+	<classpathentry kind="con" path="org.eclipse.buildship.core.gradleclasspathcontainer"/>
+	<classpathentry kind="output" path="bin/default"/>
+</classpath>
```
Comment (alexakasanjeev) [inline android/app/.classpath]: Delete this file

File: android/app/.project [hunk 4]
```diff
@@ -0,0 +1,23 @@
...
+			<name>org.eclipse.buildship.core.gradleprojectbuilder</name>
+			<arguments>
+			</arguments>
+		</buildCommand>
+	</buildSpec>
+	<natures>
+		<nature>org.eclipse.jdt.core.javanature</nature>
+		<nature>org.eclipse.buildship.core.gradleprojectnature</nature>
+	</natures>
+</projectDescription>
```
Comment (alexakasanjeev) [inline android/app/.project]: Delete this file

File: android/app/.settings/org.eclipse.buildship.core.prefs [hunk 5]
```diff
@@ -0,0 +1,2 @@
+connection.project.dir=..
+eclipse.preferences.version=1
```
Comment (alexakasanjeev) [inline android/app/.settings/org.eclipse.buildship.core.prefs]: Delete this file

**Comment by josectobar** [comment]
Im having problems seen the text in the sign up screen or adding items to cart (the button doesn't seem to be responding) I'll try again tonight and see if I can get it done.

**Comment by alexakasanjeev** [comment]
@all-contributors please add josectober for code, bug, userTesting

**Comment by alexakasanjeev** [comment]
@all-contributors please add josectobar for code, bug, userTesting

**Comment by alexakasanjeev** [comment]
> Im having problems seen the text in the sign up screen or adding items to cart (the button doesn't seem to be responding) I'll try again tonight and see if I can get it done.

I think you can use this login credentials,
> email: abc123@gmail.com
> password: wtf_changeme1

And Thank you for the screenshots, I can see there are lots of problem related to UI, i will be creating issue for each problem separately, feel free to create new issues for all the problems :)

**Pull request merged**